src/extensions/scratch3_text2speech/index.js

```diff
@@ -337,10 +350,21 @@ class Scratch3SpeakBlocks {
                     }
                 };
                 this.runtime.audioEngine.decodeSoundPlayer(sound).then(soundPlayer => {
-                    soundPlayer.connect(this.runtime.audioEngine);
+                    this._soundPlayers.set(soundPlayer.id, soundPlayer);
+
                     soundPlayer.setPlaybackRate(playbackRate);
+
+                    // Increase the volume
+                    const engine = this.runtime.audioEngine;
+                    const chain = engine.createEffectChain();
+                    chain.set('volume', 250);
```

Comment:
Can we put this magic number in a constant?

Reply:
thanks-all set!